src/docs/markdown/caddyfile/directives/metrics.md

```diff
@@ -0,0 +1,37 @@
+---
+title: metrics (Caddyfile directive)
+---
+
+# metrics
+
+Configures a Prometheus metrics exposition endpoint so the gathered metrics can
+be exposed for scraping.
+
+Note that a `/metrics` endpoint is also attached to the [admin API](/docs/api),
+which is not configurable, and is not available when the admin API is disabled.
+
+This endpoint will return metrics in the [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format)
+or, if negotiated, in the [OpenMetrics exposition format](https://pkg.go.dev/github.com/prometheus/client_golang@v1.7.1/prometheus/promhttp#HandlerOpts)
+(`application/openmetrics-text`).
+
+See also [Monitoring Caddy with Prometheus metrics](/docs/metrics).
+
+## Syntax
+
+```caddy-d
+metrics <matcher>
```

Comment:
Technically this should be:

```suggestion
metrics [<matcher>]
```

Partly for syntax highlighting reasons, but also because you could technically make a metrics subdomain that serves it on any path, because why not:

```
metrics.example.com {
	metrics
}
```

Reply:
right, I forgot about that 😅

Reply:
@francislavoie that's an interesting example - do you think it's worth adding? I'll add it now and you can let me know if it's not a good idea 😉